packages/core/src/core.types.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright IBM Corp. 2017,2020. All Rights Reserved.
```

Comment:
I'm not sure if I like the name `core.types.ts`. Adding `core.` prefix seems to be redundant as the file is already in `core` package. Otherwise, we need to add `core.` to all files?

Reply:
Well, if you have a `types.ts` file in several packages, then it's difficult to tell which file you are editing. You have to check the full path to see. The package name is not even the last entry in the path (`packages/core/src`), which makes it even more difficult. That's why I prefer to add package name prefix to common files like `types`, `keys`, `component`, and so on.

Since the code comment is many lines long, maybe it will be best to move the code to a file called something like `packages/core/src/mixin-target.ts`, WDYT?

Reply:
+1 to use `src/mixin-target.ts`.